### DISCUSSION TITLE
You are Offline! Please check your Internet Connection... Error: Getting metadata from plugin failed with error: invalid_grant

### DISCUSSION THREAD

**Discussion by renanalencar** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#318] [Q&A]
I've been facing this problem since last week. Every time I try to use the Google Assistant it displays this message on screen: 

You are Offline!
Please check your Internet Connection...
Error: Getting metadata from plugin failed with error: invalid_grant

I've already restarted the application but it keeps going with the same error message.

![Captura de tela 2021-04-17 110329](https://user-images.githubusercontent.com/776574/115115761-97620580-9f6c-11eb-8a8b-0b45c10e9d6b.png)

**Comment by Melvin-Abraham** [comment]
Please go through the [Frequently Asked Questions (FAQ)](https://github.com/Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client/wiki/Frequently-Asked-Questions-(FAQ))

**Reply by renanalencar** [reply]
Thank you.
